### PR TITLE
fix(ui): spacing inconsistencies

### PR DESCRIPTION
**Pull request by lmontero18**
ETA: Tuesday 31 3:30 pm UTC-6
resolves: #651 
test: https://holdex-venture-studio-1tr8i83o5-holdex-accelerator.vercel.app/

## Summary by CodeRabbit

- **New Features**
	- Enhanced layout and spacing for responsive design across various screen sizes.
	- Added new sections and components to improve content organization and presentation.
	- Introduced new spacing values and animations in the Tailwind CSS configuration for improved styling options.

- **Bug Fixes**
	- Adjusted margins for the contact form to improve vertical spacing. 

- **Documentation**
	- Updated comments in the navigation sections to reflect potential changes in mobile navigation elements.

### CODE REVIEW

**Comment by matthewvogas** [comment]
@lmontero18 pls before mark the PR as review follow the pr-time-tracker comment.

**Review by matthewvogas — CHANGES_REQUESTED** [review]
image
there is doble padding here


follow figma guidelines
Screenshot 2024-12-24 at 12 27 03 PM

**Comment by matthewvogas** [comment]
@lmontero18 pls resolve the margins-paddings and errores from the full page in this PR. cc @zolotokrylin

**Comment by lmontero18** [comment]
@pr-time-tracker bug commit https://github.com/holdex/holdex-venture-studio/commit/3181f54 && bug author @lmontero18

**Comment by lmontero18** [comment]
ETA log: 3:00 pm UTC-6

**Comment by lmontero18** [comment]
Not ready yet, just found something else

**Comment by zolotokrylin** [comment]
The vertical space

This branch version:
image

Production (main branch) version:
image

Is it meant to be like this?

**Comment by lmontero18** [comment]
@matthewvogas review when posible.

**Comment by matthewvogas** [comment]
@lmontero, please check figma's spaces in the head. I also made some adjustments to the design to make it more accessible, please make them in this PR. I will inform you the changes via chat so as not to spam here.

**Comment by matthewvogas** [comment]
> The vertical space
> 
> This branch version: image
> 
> Production (main branch) version: image
> 
> Is it meant to be like this?

When I made the hero a few months ago I left the hero closer to the header so it would look good, but I left the figma with a bigger space, Luis simply followed the rules to make it pixel perfect and changed it. Very good of him.

I already made the change in figma to make it shorter and look good. cc @lmontero18

**Comment by matthewvogas** [comment]
@lmontero18 updates? pls add a ETA

**Comment by lmontero18** [comment]
ETA log: 1:00 pm UTC-6

**Comment by lmontero18** [comment]
ready for review @matthewvogas

File: tailwind.config.cjs [hunk 1]
```diff
@@ -283,21 +283,32 @@ module.exports = {
...
         1.25: '0.3125rem',
         1.75: '0.4375rem',
         2.25: '0.5625rem',
+        2.5: '0.625rem',
         2.75: '0.6875rem',
         3.75: '0.9375rem',
+        4.5: '1.125rem',
         5.5: '1.375rem',
         5.6: '1.875rem',
+        5.7: '1.975rem',
```
Comment (matthewvogas) [inline tailwind.config.cjs:298]: is this really necessary to do?

File: src/routes/(pages)/index-mixins.pug [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 // hero section
 mixin section-hero-wrapper
-  .section-hero-wrapper(class="flex flex-col items-center justify-center bg-gray-100 gap-8 max-w-[1028px] w-full xs:px-4")
+  .section-hero-wrapper(class="flex flex-col items-center justify-center bg-gray-100 gap-8 mt-7.5 sm:mt-0.75 xs:mt-0.75 max-w-[1028px] w-full")
     block
 
 // KPIs section
 mixin section-kpis-wrapper
-  .section-kpis-wrapper(class="smDown:px-4 flex flex-col items-center justify-center bg-gray-100 gap-8 max-w-[1028px] w-full xs:px-4")
+  .section-kpis-wrapper(class="smDown:px-4 flex flex-col items-center justify-center bg-gray-100 gap-5.7 max-w-[1028px] w-full")
```
Comment (matthewvogas) [inline src/routes/(pages)/index-mixins.pug:14]: why 5.7? what that sizes mean?

**Review by matthewvogas — COMMENTED** [review]
left comments

File: tailwind.config.cjs (hunk 1 above)
Comment (lmontero18) [reply tailwind.config.cjs:298]: Yes, I add that size to make pixel perfect the kpi wrapper

File: src/routes/(pages)/index-mixins.pug (hunk 2 above)
Comment (lmontero18) [reply src/routes/(pages)/index-mixins.pug:14]: I dont understand that question, can you be a little more specific

File: tailwind.config.cjs (hunk 1 above)
Comment (matthewvogas) [reply tailwind.config.cjs:298]: we don't want it to be a mess in the future and duplicate measures that we can do with tailwind, can we remove that and use our own tailwind measures by doing a refactor?

**Comment by lmontero18** [comment]
ETA log: 5pm UTC-6

**Comment by matthewvogas** [comment]
@lmontero18 pls update the eta in the desc too

**Comment by matthewvogas** [comment]
@lmontero18 when update the eta pls add the day too

**Comment by lmontero18** [comment]
I noticed an inconsistency on tablets where the hero section doesn’t adapt to the borders and instead overflows. This makes the design look a bit off. My suggestion is to make the hero section slightly smaller so it adapts better to the layout.

Here the hero overflows
image

Here is the reference
image

@matthewvogas what do you think about that?

**Comment by matthewvogas** [comment]
@lmontero18  If we make it narrower, the hero will appear on three lines. Is this a problem? We should make the bottom section 'Learn' wider instead. We will change it to be wider in the future anyway (the entire site).

**Comment by lmontero18** [comment]
> @lmontero18 If we make it narrower, the hero will appear on three lines. Is this a problem? We should make the bottom section 'Learn' wider instead. We will change it to be wider in the future anyway (the entire site).

sounds good to me, so we do it in this pr or we change it in another?

**Comment by matthewvogas** [comment]
on this one, otherwise why would we merge this one?

**Comment by matthewvogas** [comment]
@lmontero18 Please provide a realistic ETA, the time limit for an open PR has been greatly exceeded, we do not want to close it or have someone else finish your PR

**Comment by lmontero18** [comment]
Yes, I'm having problems with wifi, I'll finish this asap

**Comment by matthewvogas** [comment]
@lmontero18 eta?

**Review by matthewvogas — APPROVED** [review]
It looks good, but avoid putting it as review and then as draft straight away, make sure everything is fine before putting it as review. Perfect, I'll merge it.

**Pull request merged**